docs/plugins/cloud-gce.asciidoc

```diff
@@ -1,54 +1,53 @@
-Google Compute Engine Cloud Plugin for Elasticsearch
-====================================================
+[[cloud-gce]]
+=== GCE Cloud Plugin
 
-The GCE Cloud plugin allows to use GCE API for the unicast discovery mechanism.
+The Google Compute Engine Cloud plugin uses the GCE API for unicast discovery.
 
-In order to install the plugin, run: 
+[[cloud-gce-install]]
+[float]
+==== Installation
 
-```sh
-bin/plugin install elasticsearch/elasticsearch-cloud-gce/2.5.0
-```
+This plugin can be installed using the plugin manager:
 
-You need to install a version matching your Elasticsearch version:
+[source,sh]
+----------------------------------------------------------------
+bin/plugin install elasticsearch-cloud-gce
```

Comment:
But these are the 2.0 docs right? 1.x has its own set of docs... i always think these kinds of conditionals in docs make them hard to read.

If we want to "help" people with 1.x that are reading the wrong docs, then better would be something like a disclaimer everywhere "these are the docs for version X. for other versions, go here"